src/textual/cli/tools/diagnose.py

```diff
@@ -0,0 +1,133 @@
+"""Textual CLI command code to print diagnostic information."""
+
+import os
+import sys
+import platform
+from importlib_metadata import version
+from rich.console import Console
+
+
+def _section(title: str, values: dict[str, str]) -> None:
+    """Print a collection of named values within a titled section.
+
+    Args:
+        title (str): The title for the section.
+        values (dict[str, str]): The values to print out.
+    """
+    max_name = len(max(list(values.keys()), key=len))
```

Comment:
It's a nitpick, but something like

```suggestion
    max_name = max(map(len, values.keys()))
```

Looks a bit less redundant, no?
(Ditto for max_values.)

Reply:
Actually, yeah, I think that's my preference too -- too much in the habit of staying away from `map` and friends as so much of the Python world seems to dislike "proper" functional programming. ;-)

Reply:
`max(len(k) for k in values.keys())` would also come to mind :D